src/jobs/release-lerna-packages.yml

```diff
@@ -61,6 +61,14 @@ steps:
         - ~/.cache/yarn
         - ~/.config/yarn
 
+  # - if environment looks like it is prepped to do change cases, and we're on "latest" tag
+  - when:
```

Comment:
One thing I didn't pick up on in the other PR is `sf-release npm:lerna|package:release` will not release if there is nothing to release, but we will have a change case saying there is a release... Let's file a bug to fix that later. We could do a --dryrun to see if a release will actually happen maybe... or maybe who cares if we have change cases for changes that don't actually happen? Maybe close the case as not implemented if there was nothing to release?